plutus-core/plutus-core.cabal

```diff
@@ -245,7 +245,7 @@ library
     , bimap
     , bytestring
     , cardano-crypto
-    , cardano-crypto-class        ^>=2.0.0.1
+    , cardano-crypto-class        >=2.0.0.1 && <2.2
```

Comment:
Did you verify that 2.1 works with plutus? If you don't change the index-state in cabal.project then I think the CI still uses 2.0.0.1.

Reply:
I did verify that it works in another repo (by using allow-newer)